EDI/Operation/04_ManageLineItemsFileStt.md

```diff
@@ -0,0 +1,185 @@
+[!!Exports](../UserInterface/01_Exports.md)
+[!!Manage header file settings](02_ManageHeaderFileSett.md)
+
+# Manage line item file settings
+
+The following procedures show by using an example how to specify the line item section required by your business partner. As example the D07A INVOIC message is used.
+
+For detailed information on examples how to define constant text values, string attributes, dates, and free texts, see [Manage header file settings](./02_ManageHeaderFileSett.md#).
+
+
+## Prepare data mapping for the line items
+
+Prepare the data mapping for the line items so that the relevant data for the mapping is available to retrieve the line items.
+
+#### Prerequisites
+
+- You have defined both the definitions for the header and the line items, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+- You have prepared the line item export, see [Prepare line item export](./02_ManageHeaderFileSett.md#prepare-line-item-export).
+
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the header export definition for which you want to prepare the data mapping for the line items. 
+    The *Edit definition "Definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+2. Click the *Data sources* tab.   
+    The *Data sources* tab is displayed.  
+
+    ![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesListWithEntries.png "[Data sources tab]")
+
+3. Click the data source, for which you want to map the segment for the line items.   
+    - The *Map attributes* wizard window is displayed.     
+    - In the *Destination attribute* column, a list of all EDI segment rows is displayed for which you have defined a string attribute.    
```

Comment:
Vorschlag: In the *Destination attribute* column, a list of all EDI segment rows for which you have defined a string attribute is displayed.

Reply:
Done